### ISSUE TITLE
Would love your thoughts on something.

### ISSUE THREAD

**Issue by howardroark** [whiteinge/eseth#1]
Sorry to troll you, but here I go anyway!

I saw you speak at SaltConf this past winter and was really impressed with how fluidly you spoke about Salt and how easily you could pick up on what people are trying to do with it.  I feel that the design of Salt is amazing, and understand why you are so into it.

A friend @borgstrom gave a talk about the project [StackStrap](https://github.com/strackstrap) which we had been working on.  At the time you did seem to be curious about it so I thought this would be worth while.

I'm also the guy who posted this issue... https://github.com/saltstack/salt/issues/14189

Basically I have started to abstract the "Template" ideas from our StackStrap project and build upon them elsewhere.  This way StackStrap can focus on building States that enable Vagrant based DevOps.

https://github.com/codingcoop/pollinate

I'd love any thoughts, reactions or criticism that you may have.  No pressure though!

**Comment by whiteinge** [comment]
Hello! Sorry for the slow reply -- the notification email from this repo got filtered in a strange way.

Thanks for the link to the Pollinate repo. The high-level overview in the README is intriguing. (Also, I haven't seen HJSON before. That is a nice middle-ground between the simplicity of JSON and syntax of YAML.)

I'll give this project a closer look this week.

**Comment by howardroark** [comment]
Hey, thanks for having a look at all!  I'll ping you once it is in a working condition, right now it is just in the idea stage. Maybe you would be intrigued to take it for a test drive ;)

HJSON is a great idea!  It is only available in JS form as of now. I think it would be great for writing Salt States, especially when combined with Jinja2.  Sometimes the tabbing requirements of YAML can be awkward to understand when mixed with Jijna2 templates.
